### DISCUSSION TITLE
[Bug] skywalking java agent consistently takes highly cpu spends

### DISCUSSION THREAD

**Discussion by JaydenHu6** [apache/skywalking#9527] [General]
### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

when app have lots of thread, `SkywalkingAgent-5-JVMService-produce-0` thread takes highly cpu spends

My application has 8035 threads running on an 8 core machine and the `SkywalkingAgent-5-JVMService-produce-0` thread consistently spends around 99.9% of the cpu.

<img width="1309" alt="image" src="https://user-images.githubusercontent.com/57614820/186637361-c12eff74-666a-4b68-bbe7-85cbe334a391.png">

it occurs in [JVMService](https://github.com/apache/skywalking/blob/website-docs/8.6.0/apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/jvm/JVMService.java)

<img width="1346" alt="image" src="https://user-images.githubusercontent.com/57614820/186637568-ee9f136c-4882-42b4-b775-50c0b4129984.png">

### What you expected to happen

I noticed that JVMService is scheduled every second, is it possible to schedule every 30s in default and can be configured in agent.config ?

### How to reproduce

create lots of thread in your application，and type `top -H -p ${pid}` to see cpu spends. your can use [arthas](https://arthas.aliyun.com/doc/) to find more details.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by JaydenHu6** [comment]
Besides, when my application have 3025 thread running on same machine, the `SkywalkingAgent-5-JVMService-produce-0` thread consistently spends around 16.1% of the cpu.

<img width="1078" alt="image" src="https://user-images.githubusercontent.com/57614820/186651485-a7448fa8-0691-4d80-9f5c-a171750a70c2.png">

**Reply by wu-sheng** [reply]
You could search previous discussion before asking new.
We replied this, short version, you should not have 8k threads for a Java application.

**Reply by JaydenHu6** [reply]
I agree with your point of view, but more, now the collection frequency in JVMService is too high and fixed, it should be configured

**Reply by wu-sheng** [reply]
I don't think configuration would help a lot. What do you propose to config?

**Reply by JaydenHu6** [reply]
The JVMService-produce and JVMService-consume threads are scheduled once per second. The scheduling frequency is fixed. Can the scheduling frequency be modified by parameters?

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/57614820/188297502-f8442d3c-f5e1-417f-8ecc-598fbdb7d3f1.png">

**Reply by wu-sheng** [reply]
I mean for 5k threads, even do once per 30s, you service would still not be healthy. It would be stopped when read thread information.

**Reply by wu-sheng** [reply]
If we provide a config, it seems only to expose disable option, for permanently closing this metric.
